pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -144,7 +144,22 @@ self: super: {
     then dontCheck (overrideCabal super.hakyll (drv: {
       testToolDepends = [];
     }))
-    else super.hakyll;
+    else overrideCabal super.hakyll {
+      patches = [
+        (pkgs.fetchpatch {
+          url = "https://github.com/jaspervdj/hakyll/pull/787/commits/b71955f7db1fd6532d3988be671281d1f6fcdd1d.patch";
+          sha256 = "Q9dFJpd9vmV1StjBpscDymprdF0Cdfil/tHD6c/ejBs=";
```

Comment:
These sha256 hashes look strange?  Are they base64 encoded or something?

I didn't even realize this was legal.